dsl/src/main/scala/org/http4s/dsl/io/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.http4s.dsl
+
+import cats.effect.IO
+
+package object io extends Http4sDsl[IO]
```

Comment:
Does this need to be a package object or an `object io` in `dsl`?  Package objects are hard to deprecate.  Though I guess it has a package-like feel.

Reply:
That wouldn't be a problem. It was a package object before, that's the only reason.